### PR TITLE
data: bello-olalekan.json

### PR DESCRIPTION
**Pull request by bello-olalekan**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3315"></a>

### CODE REVIEW

File: data/bello-olalekan.json [hunk 1]
```diff
@@ -2,13 +2,19 @@
...
   "type": "personal",
-  "bio": "SRE | DevOps Engineer **. Combat Sports fan",
+  "bio": "SRE | DevOps Engineer. Cloud Native and Open Source Enthusiast",
   "avatar": "https://github.com/bello-olalekan.png",
   "links": [
+     {
+      "name": "LinkedIn",
+      "url": "www.linkedin.com/in/bello-olalekan",
+      "icon": "FaLinkedin"
+    }
```
Comment (Aadarsh805) [inline data/bello-olalekan.json:12]: Missing comma here at the end of line 12

File: data/bello-olalekan.json [hunk 2]
```diff
@@ -2,13 +2,19 @@
   "name": "Bello Olalekan",
   "displayStatsPublic": true,
   "type": "personal",
-  "bio": "SRE | DevOps Engineer **. Combat Sports fan",
+  "bio": "SRE | DevOps Engineer. Cloud Native and Open Source Enthusiast",
   "avatar": "https://github.com/bello-olalekan.png",
   "links": [
+     {
+      "name": "LinkedIn",
+      "url": "www.linkedin.com/in/bello-olalekan",
```
Comment (Aadarsh805) [inline data/bello-olalekan.json:10]: fix the LinkedIn link, it should be `https://www.linkedin.com/in/bello-olalekan`

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**